CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 2.8)
+project(bootstrap-dht)
+
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/)
+
+set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -Werror ${CMAKE_CXX_FLAGS}")
+
+find_package(LibTorrentRasterbar REQUIRED)
```

Comment:
Ah yes I see. I'll fix that up.

On Fri, Jul 10, 2015 at 5:47 PM, Arvid Norberg <notifications@github.com>
wrote:

> In CMakeLists.txt
> <https://github.com/bittorrent/bootstrap-dht/pull/5#discussion_r34399336>:
>
> > @@ -0,0 +1,27 @@
> > +cmake_minimum_required(VERSION 2.8)
> > +project(bootstrap-dht)
> > +
> > +set(CMAKE_MODULE_PATH
> > +    ${CMAKE_MODULE_PATH}
> > +    ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/)
> > +
> > +set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -Werror ${CMAKE_CXX_FLAGS}")
> > +
> > +find_package(LibTorrentRasterbar REQUIRED)
>
> this adds a dependency on libtorrent, doesn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bittorrent/bootstrap-dht/pull/5/files#r34399336>.
>